Formula/vtk@8.2.rb

```diff
@@ -0,0 +1,101 @@
+class VtkAT82 < Formula
```

Comment:
Please indicate how this would fill our criteria for versioned formulas (https://docs.brew.sh/Versions) and why we might want to ship an older version?

Reply:
I have updated the commit message that introduces this formula to explain my motivation for its existence (and I will paste it here as well). Is this sufficient?

```
    In addition to many added features and bug fixes (see
    https://blog.kitware.com/vtk-9-0-0-available-for-download/ for more
    details), VTK's move from 8.2 to 9.0 came with a redesign of its build
    logic, including a renaming and reorganization of its build targets.
    Dependent projects will therefore need to modify their build logic to
    accommodate VTK's new build system. Until the latest version of VTK is
    sufficiently well adopted by the open-source community, it is necessary
    to support both the current release and the release from the last major
    version.
```

Reply:
How about 
> Upstream should have a release branch for each formula version, and release security updates for each version when necessary. 

And

> Versioned formulae submitted should be expected to be used by a large number of people.￼

Reply:
> Upstream should have a release branch for each formula version, and release security updates for each version when necessary.

VTK has a tag for v8.2.0. If necessary, the VTK community will accept patch updates for this version to address security concerns (and has done so in the past). In general, though, the VTK community does not have the bandwidth necessary to support multiple release branches. 

> Versioned formulae submitted should be expected to be used by a large number of people.￼

VTK v8.2.0, currently provided by the recipe `vtk`, is #808 on your list of [Formula Install On Request Events](https://formulae.brew.sh/analytics/install-on-request/365d/). This puts VTK v8.2.0 well above your threshold for being in the top 3,000 formulae.

Reply:
> VTK has a tag for v8.2.0. If necessary, the VTK community will accept patch updates for this version to address security concerns (and has done so in the past). In general, though, the VTK community does not have the bandwidth necessary to support multiple release branches.

In that case we can't really accept this as a version.

Reply:
> In that case we can't really accept this as a version.

In that case, what is the correct way to proceed? VTK is currently at v9.0.1, and at least one formula in homebrew-core depends on v8.2. VTK v8.2 is by no means unsupported, but semantically we do not have a branch for it (instead, we have a tag). We support patch releases for v8.2 in the event of security-related updates. If this is not sufficient for adding a version formula, then what is the appropriate way to provide the latest VTK release via homebrew without breaking downstream projects?